### ISSUE TITLE
Swift 4 AlamofireObjectMapper Dependency

### ISSUE THREAD

**Issue by RudyB** [Daltron/Spartan#4]
Hello,
Just writing to say that the Podfile spec should probably be updated for AlamofireObjectMapper 5 so that users can use swift 4

```
[!] Unable to satisfy the following requirements:

- `AlamofireObjectMapper (from `https://github.com/tristanhimmelman/AlamofireObjectMapper.git`, branch `swift-4`)` required by `Podfile`
- `AlamofireObjectMapper (= 5.0.0)` required by `Podfile.lock`
- `AlamofireObjectMapper (~> 4.0.1)` required by `Spartan (1.0)`
```

**Comment by Daltron** [comment]
Hi @RudyB! 😄 AlamofireObjectMapper doesn't yet have a swift 4 release that can be used by pods who use AlamofireObjectMapper as a dependency. [You can follow this issue](https://github.com/tristanhimmelman/AlamofireObjectMapper/issues/215) as I have let the author know that a swift 4 release would be greatly appreciated. In the meantime, you can still use Spartan within a Swift 4 project as Swift 3.x and Swift 4 are completely compatible with each other. Just include this at the bottom of your podfile:

```ruby
post_install do |installer|
    installer.pods_project.targets.each do |target|
        swift3_projects = ['Spartan']
        if swift3_projects.include? target.name
            target.build_configurations.each do |config|
                config.build_settings['SWIFT_VERSION'] = '3.2'
            end
        end
    end
end
```

**Comment by RudyB** [comment]
@Daltron I saw that you were pretty active on their issue board. I am also patiently awaiting a swift 4 release from them.

I added that code to the bottom of my pod file and I am still getting the Object Mapper Build Issues regarding the `toIntMax`

Did I do something dumb?


A copy of my pod file:

```
target 'Tempo' do

  use_frameworks!
  platform :ios, '10.3'
  # Pods for Tempo
  pod 'SpotifyLogin'
  pod 'Spartan'

  target 'TempoTests' do
    inherit! :search_paths
    # Pods for testing
  end

  target 'TempoUITests' do
    inherit! :search_paths
    # Pods for testing
  end

end

post_install do |installer|
    installer.pods_project.targets.each do |target|
        swift3_projects = ['Spartan']
        if swift3_projects.include? target.name
            target.build_configurations.each do |config|
                config.build_settings['SWIFT_VERSION'] = '3.2'
            end
        end
    end
end

```

**Comment by Daltron** [comment]
@RudyB Did you rerun the `pod install` command? I forgot to mention that that is needed in order for the changes to take affect! 😄

**Comment by RudyB** [comment]
I absolutely did. I also cleaned my project and build folder.

**Comment by Daltron** [comment]
@RudyB Can you take a screenshot of the errors Xcode is reporting please?

**Comment by RudyB** [comment]
[Pod Install Output](https://www.dropbox.com/s/2gao30d8zf7s2fh/Screenshot%202017-09-28%2011.31.06.png?dl=0)
[Xcode Output](https://www.dropbox.com/s/1ao88qrb1izpkk9/Screenshot%202017-09-28%2011.30.35.png?dl=0)

**Comment by Daltron** [comment]
@RudyB Ahhh I'm sorry, you need to also specify 'AlamofireObjectMapper' and 'Alamofire' in the `swift3_projects` variable. I've updated my original comment to include these two pods.

**Comment by RudyB** [comment]
I had to explicitly add `ObjectMapper` to the swift3_projects variable as well in order to get it to work. Thank you so much for all the help. You may want to add that to your code snippet

**Comment by Daltron** [comment]
Glad to hear! 🎉

**Comment by RudyB** [comment]
AlamofireObjectMapper 4.1.0 was released today FYI

**Comment by Daltron** [comment]
@RudyB Swift 4 support is now available in [v1.1.0](https://github.com/Daltron/Spartan/releases/tag/1.1.0) release! 🎉

**Comment by RudyB** [comment]
@Daltron Thank you for your hard work.
